media-libs/libsmf/libsmf-1.3.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="A C library for handling SMF files"
+DESCRIPTION="LibSMF is a BSD-licensed C library for handling \
+SMF (*.mid) files. It transparently handles conversions between time \
+and pulses, tempo map handling etc. The only dependencies are C \
+compiler and glib. Full API documentation and examples are included"
+HOMEPAGE="https://sourceforge.net/projects/libsmf/"
+COPYRIGHT="2007, 2008 Edward Tomasz Napieraa \
```

Comment:
You should probably replace "2007, 2008" by "2007-2008".
(I haven't checked, however, if that range is correct.)